common/logger.py

```diff
@@ -15,6 +16,13 @@ def add_logger(name, level):
         formatter = logging.Formatter('%(message)s')
 
         log_file = join(path, f'{name}.log')
+        
+        if not exists(path):
+            subprocess.run(["mkdir", path])
```

Comment:
Use `os.mkdir` instead of calling `subprocess`. It makes this project automatically platform dependent. Let's avoid it.